cli/cmd/plugin/ui/web/tanzu-ui/src/shared/components/ConnectionNotification/ConnectionNotification.tsx

```diff
@@ -0,0 +1,41 @@
+// React imports
+import React from 'react';
+// Library imports
+import { CdsAlert, CdsAlertGroup } from '@cds/react/alert';
+// App imports
+import './ConnectionNotification.scss';
+
+export function ConnectionNotification(
+    connected: boolean,
```

Comment:
I feel like we could refactor this a bit. to be more dynamic and extensible.
I'm thinking instead of passing multiple params for connected, connecting,
we could define one config of type
interface IConnectionNotification {
status: 'connected' | 'connecting' | 'failed'
message: 'sample message',
}

and pass in the updated config object.  This could be big change. we could address it later point of time.

Reply:
I agree with you in principle, and it definitely ran through my mind that the parameter list is awkward. Happy to discuss options, or leave it for a later refactor.